doc/index.rst

```diff
@@ -5,10 +5,9 @@
 
 .. container:: index-paragraph
 
-    Nilearn is a Python module for **fast and easy statistical learning on
-    NeuroImaging** data.
+    Nilearn is a Python module to enable **efficient statistical learning and analysis of brain volumes** through **approachable methods and tutorials**.
```

Comment:
and what about surfaces :smile:

Reply:
They should be properly scrubbed and cleaned regularly?

Reply:
Should I add surface as well? I didn't think Nilearn did too much with surfaces, only some basic stuff.